dist/Dockerfile.e2e/Dockefile

```diff
@@ -0,0 +1,15 @@
+FROM centos:7
+
+ENV HOME="/root"
+
+# Get oc CLI
+RUN mkdir -p ${HOME}/bin && \
+    curl https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/4.3.0-0.nightly-2019-11-13-233341/openshift-client-linux-4.3.0-0.nightly-2019-11-13-233341.tar.gz 2>/dev/null | tar xzf - -C /tmp/bin/ oc
+ENV PATH="${PATH}:${HOME}/bin"
+
+RUN mkdir -p "${HOME}/cincinnati"
+WORKDIR ${HOME}/cincinnati
```

Comment:
nit: moving these eariler will mean we can preserve those layers when we bump `oc`.  Unlikely to matter, but I like having more-stable steps earlier in the `Dockerfile` when we can do that without making things awkward.

Reply:
Sounds like a good idea, implemented